### ISSUE TITLE
Maximum call stack size exceeded at RegExp.toString when using date mask 'mm/dd/yyyy' and 'min' property

### ISSUE THREAD

**Issue by Techn1c4l** [RobinHerbots/jquery.inputmask#2754]
This issue occurs only when certain conditions are met:
1. Alias set to "datetime".
2. Format is set to "mm/dd/yyyy" or similar beginning with "mm".
3. "min" property is set to a date which month is greater than 9.
4. You attempt to enter anything than "1", e.g. "2" or "3".

Those values are not valid, so I shouldn't be able to input them. Instead it hangs for a second and throws an error with following description:

```
inputmask.js:3180 Uncaught RangeError: Maximum call stack size exceeded
    at RegExp.toString (<anonymous>)
    at f (inputmask.js:3180:216)
    at inputmask.js:3282:61
    at c2 (inputmask.js:3302:33)
    at k (inputmask.js:3321:37)
    at y.p (inputmask.js:3344:33)
    at y.u (inputmask.js:3172:80)
    at y.c (inputmask.js:3159:42)
    at y.s (inputmask.js:3148:64)
    at t2.getMaskTemplate (inputmask.js:3112:40)
```

If you set a minimum month to 9 or less, the issue is not present.

Example page to reproduce:

```
<!DOCTYPE html>
<html lang="en-US">
<head>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.0/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.inputmask/5.0.8/jquery.inputmask.min.js"></script>
</head>
<body>
<form>
	<input type="text" id="field" ><label class="form-label" for="field"> Try to enter a month beginning with anything else than '1'</label>
</form>


<script>
window.onload = function () {
	jQuery("#field").inputmask( {"alias":"datetime","inputFormat":"mm/dd/yyyy","min":"10/25/2023","max":"11/25/2023"});
};
</script>
</body>
```
</html>

**Comment by RobinHerbots** [comment]
@Techn1c4l ,

This seems fixed in v5.0.9.

Can you confirm.

**Comment by Techn1c4l** [comment]
> @Techn1c4l ,
> 
> This seems fixed in v5.0.9.
> 
> Can you confirm.

Confirmed, works fine on 5.0.9.
